src/mono/mono/mini/mini-x86.h

```diff
@@ -154,27 +154,18 @@ typedef struct {
 
 #define MONO_CONTEXT_SET_LLVM_EXC_REG(ctx, exc) do { (ctx)->eax = (gsize)exc; } while (0)
 
-#ifdef _MSC_VER
-
-#define MONO_INIT_CONTEXT_FROM_FUNC(ctx, start_func) do { \
-    unsigned int stackptr; \
-    { \
-	   __asm mov stackptr, ebp \
-    } \
-	MONO_CONTEXT_SET_IP ((ctx), (start_func)); \
-	MONO_CONTEXT_SET_BP ((ctx), stackptr); \
-	MONO_CONTEXT_SET_SP ((ctx), stackptr); \
-} while (0)
-
-#else
+#if defined(HOST_WIN32)
```

Comment:
Believe we could do something similar in mini-amd64.h as well.

Reply:
Falling into cleanup/alignment category.

Reply:
The notes on the non-Windows case imply problems using __builtin_frame_address with gcc() (clearly only an issue on amd64, since we use it on other arches) so I'd be nervous about aligning the Windows case there. I feel changes for the non-Windows case on amd64 (and any alignment between the msvc/gcc case) should be part of another PR